pkg/settings/settings.go

```diff
@@ -27,6 +27,7 @@ var (
 	UIIndex                = NewSetting("ui-index", "https://releases.rancher.com/harvester-ui/latest/index.html")
 	UIPath                 = NewSetting("ui-path", "/usr/share/rancher/harvester")
 	APIUISource            = NewSetting("api-ui-source", "auto") // Options are 'auto', 'external' or 'bundled'
+	VolumeSnapshotClass    = NewSetting("volume-snapshot-class", "longhorn")
```

Comment:
How are we gonna use this setting?
Is there any affection if we have one storageclass per image?

Reply:
this is used by the CSI volume snapshot feature, make this configurable would benefit users using another storage driver in the future.